easybuild/easyconfigs/f/futile/futile_intel_bug.patch

```diff
@@ -0,0 +1,47 @@
+diff -rup futile-1.8.3.orig/flib/fpython.f90 futile-1.8.3/flib/fpython.f90
```

Comment:
Could you please add a short description what the patch does, and note who the author is?

Reply:
@micaeljtoliveira Ping on this?

Reply:
@boegel Thanks for the ping. For some reason I missed this comment. I've now added a short description and the author to the patch.